src/scene/mesh.js

```diff
@@ -202,10 +202,11 @@ class Mesh extends RefCountedObject {
      *
      * - `base` is the offset of the first index or vertex to dispatch in the draw call.
      * - `count` is the number of indices or vertices to dispatch in the draw call.
+     * - `baseVertex` is the number added to each index value before indexing into the vertex buffers. (available only for WebGPU and Batching)
      * - `indexed` specifies whether to interpret the primitive as indexed, thereby using the
      * currently set index buffer.
      *
-     * @type {{type: number, base: number, count: number, indexed?: boolean}[]}
+     * @type {{type: number, base: number, count: number, baseVertex?: number, indexed?: boolean}[]}
```

Comment:
we should initialize it to 0 here as well, instead of leaving it undefined

Reply:
In most cases, this parameter will be of little use, and there is no need to store it in memory for each primitive.